miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/Type.kt

```diff
@@ -21,3 +22,9 @@ internal enum class AdType(val value: Int) {
     INTERSTITIAL(0),
     REWARDED(1),
 }
+
+internal enum class ScreenAction(val action: String) {
+    LOCK_PORTRAIT("rakuten.miniapp.screen.lock_portrait"),
```

Comment:
Can we write the postfix as `LOCK_PORTRAIT` instead of `lock_portrait`. We followed the same convention in custom permission as well.